app/components/satis/dropdown/component.html.slim

```diff
@@ -1,5 +1,5 @@
 div.satis-dropdown data-action="keydown->satis-dropdown#dispatch" data-controller="satis-dropdown" data-satis-dropdown-page-size-value=@page_size data-satis-dropdown-url-value=url data-satis-dropdown-url-params-value=(options[:url_params]||{}).to_json data-satis-dropdown-chain-to-value=@chain_to data-satis-dropdown-free-text-value=@free_text data-satis-dropdown-needs-exact-match-value="#{@needs_exact_match ? 'true' : 'false'}" data-satis-dropdown-is-multiple-value="#{options[:input_html][:multiple] || false}"
-  - selection = [nil]
+  - selection = []
```

Comment:
This is about the fourth time we change this from [] to " " to [] to [nil] and now [] again? It seems we've gone full circle. Twice. 🤪
We need tests, check the system tests in OMS please and verify if this specific case is dealt with.